index.html

```diff
@@ -11,169 +11,223 @@
+                          <p><img src="image/Img-Placeholder.png" alt="image of project" class="ph-image"></p>
+                          
 
-        <div class="workedOn" id="work">
-          <div class="project-flex">
-            <div class="flex-item"><img src="image/Img-Placeholder.png" alt="image of project"></div>
-            <div class="flex-item"><h2>Multi-Post Story</h2></div>
-            <p class="flex-item holdtext">
-              A daily selection of privately personalized reads; 
-              no accounts or sign-ups required. 
-              has been the industry's standard dummy text ever since the 1500s, 
-              when an unknown printer took a standard dummy text.
-            </p>
-            <ul class="flex-item navlang-flex">
-              <li><a href="#" class="subflex-item artag">CSS</a></li>
-              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
-              <li><a href="#" class="subflex-item artag">html</a></li>
-              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
-              <li><a href="#" class="subflex-item artag">bootstrap</a></li>
-              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
-              <li><a href="#" class="subflex-item artag">Ruby</a></li>
-              </ul>
-            <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
-          </div>
+                      </div>
+                      <div class="project-text-nav">
+                          <h2 class="hh2">Multi-Post Story</h2>
+                          <p class="flex-item holdtext">
+                              A daily selection of privately personalized reads; 
+                              no accounts or sign-ups required. 
+                              has been the industry's standard dummy text ever since the 1500s, 
+                              when an unknown printer took a standard dummy text.
+                          </p>
+                          <ul class="flex-item navlang-flex">
+                              <li><a href="#" class="subflex-item artag">CSS</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">html</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">bootstrap</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">Ruby</a></li>
+                              </ul>
+                          <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
```

Comment:
Kindly implement button interactions for all the buttons according to the Figma design 🤗

![image](https://user-images.githubusercontent.com/73714615/187038919-13a33195-7fa4-4f3d-9672-3b592c26b3b0.png)